DESCRIPTION

```diff
@@ -57,11 +57,13 @@ Imports:
     plotly,
     NHSRplotthedots,
     reshape2,
-    NHSRtheme (>= 0.1.0)
+    NHSRtheme (>= 0.1.0),
+    fresh (>= 0.2.0)
 Remotes:  
     CDU-data-science-team/nottshcMethods,
     CDU-data-science-team/experienceAnalysis,
-    nhs-r-community/NHSRtheme
+    nhs-r-community/NHSRtheme,
+    dreamRs/fresh
```

Comment:
My only comment on this PR is can we use the CRAN version of the package instead?

Incidentally, I don't think either nottshcMethods or experienceAnalysis are actually in use any more in the package

Reply:
ok. switched to the CRAN location. https://github.com/CDU-data-science-team/experiencesdashboard/pull/137/commits/9d52f30244fb242ad03d03b44c0e45fd219fd32c
nottshcMethods or experienceAnalysis are actually not in use but I haven't removed them because a large part of the repo still needs removing. I have an open issue for this